src/app/locationForm/description/OrganizationDescription.jsx

```diff
@@ -8,23 +8,23 @@ import FormEdit from '../common/FormEdit';
 import FormView from '../common/FormView';
 import { compose, withProps } from 'recompose';
 
-const LocationDescriptionEdit = compose(
+const OrganizationDescriptionEdit = compose(
   withProps({
-    headerText : 'Please describe this location',
+    headerText : 'How would you describe this organization?',
     placeholderText : 'Enter a description of the location',
   })
 )(props => <FormEdit {...props} />)
 
-const LocationDescriptionView = compose(
+const OrganizationDescriptionView = compose(
   withProps({
     topText : 'LOCATION DESCRIPTION',
```

Comment:
The topText and placeholderText are still the old ones for location description.

Reply:
Thanks, I didn't see that. Fixed in 24f4b92d804edef6fca11bc2b9baaf9254c8d56a